plugin/restore/mysql/mysql.go

```diff
@@ -49,14 +49,14 @@ func New(driver *mysql.Driver, instance *mysqlutil.Instance, connCfg db.Connecti
 }
 
 // replayBinlog restores the database using incremental backup in time range of [config.Start, config.End).
-func replayBinlog(ctx context.Context, config api.BinlogInfo) error {
+func replayBinlog(ctx context.Context, config api.BinlogInfo, saveDir string) error {
```

Comment:
Please use more self-explanatory var names `binlogDir` (and be consistent with other occurrences).

Reply:
SG